src/CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ if (UNIX AND ENABLE_SINGLE_FILES_WERROR)
   # Enables stricter compiler checks on a file-by-file basis
   # which allows us to migrate piecemeal
   set_source_files_properties(proto_conversions.cc PROPERTIES COMPILE_FLAGS "-Wall -Werror")
-endif (UNIX)
+endif (UNIX AND ENABLE_SINGLE_FILES_WERROR)
```

Comment:
why not make all of these just `endif ()`? indentation is surely enough to match these up

Reply:
see here: https://cmake.org/cmake/help/latest/command/endif.html

> The optional <condition> argument is supported **for backward compatibility only**. If used it must be a verbatim repeat of the argument of the opening if clause.

I highly suggest we dont put anything in the condition as its meaningless and extra crap to maintain

Reply:
Fully agree. I didn't know that was an option!